### ISSUE TITLE
Flushbar does not meet WCAG 2.1 Status Message Criterion 4.1.3

### ISSUE THREAD

**Issue by aaronkelton** [cmdrootaccess/another-flushbar#79]
The criteria for status messages at https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html indicates that these flushbars should announce their contents when displayed on-screen (using phone features like iOS VoiceOver or Android TalkBack). At present, I do not believe these flushbars meet the accessibility criteria.

Repo with video recording demonstration in readme: https://github.com/aaronkelton/flushbar-accessibility-status-message

Is another_flushbar accessible out of the box, or are these criteria something worth supporting? At the moment I am considering swapping out the Flushbar widgets for something else due to a new client's requirements to pass an accessibility audit.

**Comment by cmdrootaccess** [comment]
hello,
currently this is not supported but i can take a look at it
